TemplateApplication/Helper/ProcessInfo+PreviewSimulator.swift

```diff
@@ -0,0 +1,15 @@
+//
+// This source file is part of the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import Foundation
+
```

Comment:
Slightly nitpicky, but would be great to keep that convention around for all projects as noted in the Onboarding PR: https://github.com/StanfordSpezi/SpeziOnboarding/pull/10#discussion_r1285266106. 
```suggestion


```

Applies to some other Swift files as well 👍